src/components/project/ProjectsImportDialog.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+  <div class="ProjectsLoadDialog">
+    <v-dialog v-model="state.dialog" max-width="1024">
+      <v-card>
+        <v-card-title v-text="'Import projects'" />
+        <v-card-subtitle v-text="'Select a source'" />
+
+        <v-card-text>
+          <v-row class="mb-1">
+            <v-col cols="2">
+              <v-btn-toggle
+                dark
+                dense
+                mandatory
+                v-model="state.source"
+              >
```

Comment:
I receive the suggestion
```suggestion
              <v-btn-toggle dark dense mandatory v-model="state.source">
```
by ESLint/Prettier. Could you check that?